UPGRADE-1.x.md

```diff
@@ -0,0 +1,10 @@
+UPGRADE 1.x
+===========
+
+UPGRADE FROM 1.4 to 1.5
+=======================
+
+## Changes in directory structure
+
+The `lib` directory has been renamed to the more standard `src`.
+Also the directory structure of `src` and `test` no longer follows `PSR-0`, but `PSR-4`.
```

Comment:
New line before this sentence.

Maybe also add a line telling this **should** not change anything because the library is aim to be used with composer only?